src/app/encrypt/encrypt.hpp

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include <iomanip>
+#include <openssl/evp.h>
+#include <sstream>
+#include <string>
```

Comment:
In my opinion none of this headers is needed in hpp file and they should be moved to cpp file.
Here are some reasons:
1. Minimize dependencies: Including only what’s necessary in your header files reduces dependencies. This can lead to faster build times and fewer issues with circular dependencies

2. Encapsulation: Header files are often considered the public interface to your classes. By including only what’s necessary, you avoid exposing implementation details

3. Avoid multiple inclusions: Each .cpp file is generally built individually, which means a .h file will be re-parsed for each .cpp file you compile